recipes/kanidm-clients/meta.yaml

```diff
@@ -0,0 +1,53 @@
+{% set name = "kanidm-clients" %}
+{% set version = "v1.1.0rc16" %}
```

Comment:
Either set this to a release or set the channel_target to a dev/rc channel.

Reply:
Despite this being a 1.1.0 rc there is no non-rc release yet.
Kanidm went from 1.0.0rc3 to 1.1.0-alpha without a 1.0.0 release